### DISCUSSION TITLE
Problems with vulnerabilities for suse server 12.1

### DISCUSSION THREAD

**Discussion by natanael995** [wazuh/wazuh#22327] [Q&A]
Community, I'm having problems loading the vulnerability detection module to the suse server, I don't have access to the internet and I'm doing it offline as the documentation requires. However, the CVE's that load are up to 2020, when opening the xml file I saw that they are updated until 2024, could you help me with this issue?

**Comment by MiguelazoDS** [comment]
Hello @natanael995, 

Sorry for the delay, if I am not getting you wrong, you're not getting CVEs newer than 2020. I did a quick test and I was able to get all of them. 

![image](https://github.com/wazuh/wazuh/assets/13010397/200a940e-ed34-4f23-827e-839c0cec6585)

This is the configuration I used 

```xml
    <!-- SUSE OS vulnerabilities -->
    <provider name="suse">
      <enabled>yes</enabled>
      <os path="/home/vagrant/suse.linux.enterprise.server.12.xml.gz">12-server</os>
      <update_interval>1h</update_interval>
    </provider>
```

Did I understand well the issue you're having?
